django_ses/__init__.py

```diff
@@ -227,6 +227,7 @@ def _update_throttling(self):
         # Since I'm not sure how Amazon determines at exactly what
         # point to throttle, better be safe than sorry and let in, say,
         # half of the allowed rate.
+        recent_send_times.append(now)
```

Comment:
Is the comment on line 211 still valid with this change?

What did you use this for?